app/webpacker/controllers/searchbox_controller.js

```diff
@@ -64,8 +74,15 @@ export default class extends Controller {
   }
 
   onConfirm(chosen) {
-    if (chosen && chosen.link)
+    if (chosen && chosen.link) {
+      if (this.analyticsSubmitter) {
+        this.clearAnalyticsSubmitter()
+
+        this.sendToAnalytics()
```

Comment:
`clearAnalyticsSubmitter()` is also called at the top of `sendToAnalytics()`, I think it makes more sense there than here?